git/commonGitCommands.txt

```diff
@@ -0,0 +1,22 @@
+- A and B are friends and team members working in a project
+
+- A creates a project (group of files), he is the owner
+- A uploads it to github.com (original copy) named it as "main" & shared with team(A & B)
+- B downloads/clone it to his system
+-  creates a copy(checkout) and name it "feature"
+    git checkout -b feature
```

Comment:
Hey there, the changes are good. However, please rename the variables for so & so reason.

Reply:
@thabungm ok great!!